docs/sla.md

```diff
@@ -0,0 +1,93 @@
+# Service Level Agreement(SLA)
+
+Service Level Agreement is available to Premium Customers only.
+
+
+## Definitions
+
+* "Self-service Console" provides access to a self-service console at https://console.cdnetworks.com. The console provides a way to configure the service through the Web UI or API. 
+* "Self-service Documentation" is a self-service documentation portal at https://docs.cdnetworks.com and can be accessed at any time.
+* "Email Support" is a way to obtain help through one of the dedicated email addresses that can be found at https://www.cdnetworks.com/contact
+* "Phone Support" is a way to obtain help through a dedicated phone line. Toll-free phone numbers can be found at https://www.cdnetworks.com/contact.
+* "Direct Tickets Support" is a way to obtain help through a dedicated ticket tracker, where all requests can be managed and handled easily.
+* "Business Hours" are working hours (9AM to 6PM, local time) in California, London, and Beijing.
+* "Business Days" are Monday through Friday at the customer location. 
+* "Urgent requests" are for issues that impact 50% or more of the customer’s users.
```

Comment:
The capitalization of these phrases seem arbitrary.

Reply:
We should be consistent with capitalization. Either capitalize "requests" for "Urgent requests" or capitalize only the first letter in the terms surrounds by quotes.
You can remove the comma between "6PM" and "local."
For "Business Days," do we want to mention any holidays when we are not available, such as Xmas?
I suggest changing "Direct Tickets Support" to "Direct Ticket Support" (remove the s from "Tickets"),